simpleContentNft

```diff
@@ -126,10 +126,10 @@ contract simplecontentNft is Context, ERC165, IERC721, IERC721Metadata {
         bool _asm
     ) {
 
-        string[] memory pi = _contents[tokenId]._productInfo;
-        string[] memory cds = _contents[tokenId]._contractDocuments;
-        uint256[] memory crfr = _contents[tokenId]._copyrightFeeRatio;
-        bool asm = _contents[tokenId]._allowSecondaryMerket;
+        string[] memory pi = _contents[tokenId]._info;

```

Comment:
The variable name "pi" does not indicate the entity.
The variable name should be "info".
"cds", "crfr", and "asm" are also the same review comments.